### DISCUSSION TITLE
Simple Question about  "Maximum number of saved sessions"

### DISCUSSION THREAD

**Discussion by bogdan-h** [sienori/Tab-Session-Manager#902] [Q&A]
In the "Settings" what exactly does it mean "Maximum number of saved sessions (in this way)." ?
This is mentioned for :
- Save the session regularly
- Save the session when window was closed
- Save the session when exiting browser

Does it means that if we are setting the number to : let's say 10.  After 10 sessions, the older one is deleted in order to add a new one ?

What are the _usual_ numbers that people are using for these settings : save regularly, windows closed and browser exit ?
This in the context where you might need to discharge sessions that are older than 9 (or 12) months ?

Thank you in advance, 
Bogdan.

**Comment by rediffusion** [comment]
It saves maximum 10 times, after that i guess it stops.

**Comment by abdo2048** [comment]
Could we get a clear explanation of it? because what is written here https://github.com/sienori/Tab-Session-Manager/issues/559 gives me mixing feelings!
